src/Link.js

```diff
@@ -1,10 +1,12 @@
 import styled, { css } from 'react-emotion';
 import PropTypes from 'prop-types';
+import { color } from 'styled-system';
```

Comment:
Do we really want to expose color to the world like this? It seems like something that would conflict with our variants.

Reply:
I needed to change the link color for Action Items in our Shot Card example.